extensions/amp-truncate-text/1.0/truncate-text.type.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @externs */
+
+/** @const */
+var TruncateTextDef = {};
+
+/**
+ * @typedef {{
+ *   slotCollapsed: (?PreactDef.VNode|undefined),
```

Comment:
I think all of these are all `PreactDef.Renderable` instead of `PreactDef.VNode`

Reply:
Done